.cfnlintrc

```diff
@@ -1,2 +1,3 @@
 ignore_checks:
   - W3002
+  - W2001
```

Comment:
we should not need this - if it says there are issues, fix the issues, dont ignore them